indico/modules/rb_new/controllers/user/landing.py

```diff
@@ -16,10 +16,51 @@
 
 from __future__ import unicode_literals
 
+import dateutil
+from flask import request
+
+from indico.core.db import db
+from indico.modules.rb import rb_settings
 from indico.modules.rb.controllers import RHRoomBookingBase
+from indico.modules.rb.models.rooms import Room
 from indico.modules.rb_new.views.base import WPRoomBookingBase
+from indico.util.string import natural_sort_key
 
 
 class RHRoomBookingLanding(RHRoomBookingBase):
     def _process(self):
         return WPRoomBookingBase.display('room_booking.html')
+
+
+class RHRoomBookingSearch(RHRoomBookingBase):
+    def _process(self):
+        from indico.modules.rb.schemas import rooms_schema
+
+        min_capacity = int(request.args['capacity']) if 'capacity' in request.args else None
+        room_name = request.args.get('title')
+        start_dt = dateutil.parser.parse(request.args['start_dt'])
+        end_dt = dateutil.parser.parse(request.args['end_dt'])
+        repeat_frequency = int(request.args.get('repeat_frequency', 0))
```

Comment:
Shouldn't this be a `RepeatFrequency` value? It will end up being an int, sure, but maybe it's nicer if we accept a string here?

Reply:
Can be as well.